# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/wgpu-types/.cargo-checksum.json
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/wgpu-types/.cargo-checksum.json b/third_party/rust/wgpu-types/.cargo-checksum.json
new file mode 100644
--- /dev/null
+++ b/third_party/rust/wgpu-types/.cargo-checksum.json
@@ -0,0 +1,1 @@
+{"files":{"Cargo.toml":"6d77eb29f5ae7d0d1ad662b81ed91db3566c52cbe202a0adf0d722cb383360ab","src/lib.rs":"52ab2cf4329fba5a33122f7a18113b9436282f991caeac45b43583e6fac20bf2"},"package":null}
\ No newline at end of file